src/routes/index.ts

```diff
@@ -533,6 +554,22 @@ router.route(config.UPDATE_MANAGING_OFFICER_URL + config.ID)
   .post(...validator.updateManagingOfficerIndividual, checkValidations, updateManagingOfficerIndividual.update);
 router.get(config.UPDATE_MANAGING_OFFICER_URL + config.REMOVE + config.ID, authentication, navigation.hasUpdatePresenter, updateManagingOfficerIndividual.remove);
 
+router.route(config.UPDATE_REVIEW_INDIVIDUAL_MANAGING_OFFICER_URL)
+  .all(
+    authentication,
+    companyAuthentication,
+    navigation.hasUpdatePresenter
+  )
+  .get(updateReviewIndividualManagingOfficer.get)
+  .post(...validator.updateBeneficialOwnerAndManagingOfficerType, checkValidations, updateReviewIndividualManagingOfficer.post);
+
+router.route(config.UPDATE_REVIEW_BENEFICIAL_OWNER_GOV_URL + config.UPDATE_REVIEW_OWNERS_PARAMS)
```

Comment:
UPDATE_REVIEW_BENEFICIAL_OWNER_GOV_URL doesn't look right?

Reply:
I think this is defined above already

Reply:
Removed